src/blockTypes/nixosConfigurations.nix

```diff
@@ -17,7 +17,40 @@
     type = "nixosConfiguration";
     # nixosGenerator's actions?
     # microvm action?
-    # nixos-rebuild action?
+    actions = {
+      currentSystem,
+      fragment,
+      fragmentRelPath,
+      target,
+      inputs,
+    }: let
+      getString = o: (l.elemAt (l.splitString ["/"] fragmentRelPath) o);
+      host = (getString 0) + "-" + (getString 2);
+      dc = getString 1;
+      bin = ''
+        bin=$(nix build .#${dc}.${host}.system --no-link --print-out-paths)/sw/bin
+        export PATH=$bin:$PATH
+      '';
+    in (l.attrsets.mapAttrsToList (name: description: (mkCommand currentSystem {
+        inherit name description;
+        command =
+          bin
+          + "${
+            l.optionalString (l.elem name ["switch" "boot" "test" "dry-activate"]) "run0 --setenv=PATH=\"$PATH\" "
+          }nixos-rebuild ${name} --flake . -L --show-trace";
```

Comment:
Should we pass the `-L` and `--show-trace` flags here?

I prefer that, since it's more verbose.
What do you think.

Reply:
We could add a `$@` to pass along any user supplied flags. That would be a bit more consistent with the way other block types handle things.